### PR TITLE
Added files for verifying [Policies] SDL.ActivateApp from HMI and 'isAppPermissionsRevoked' parameter in the response

### PR DESCRIPTION
**Pull request by aarotar**

### CODE REVIEW

**Comment by aarotar** [comment]
@vvvakulenko  please review

**Comment by aarotar** [comment]
@aderiabin please review

**Comment by aarotar** [comment]
@vvvakulenko please review

**Comment by aarotar** [comment]
@vvvakulenko, @aderiabin please review

**Comment by vvvakulenko** [comment]
@maria-mih Please review

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+local Preconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+function Test:CloseConnection()
```
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: Please rename functions in this section to contain Precondition_ e.g. 
function Test:Precondition_CloseConnection()

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+	self.mobileConnection:Close()
+	commonTestCases:DelayedExp(3000)	
+end
+
+Preconditions:BackupFile("sdl_preloaded_pt.json")
+testCasesForPolicyTable:Precondition_updatePolicy_By_overwriting_preloaded_pt("files/GroupsForApp_preloaded_pt.json")
+
+function Test:ConnectDevice()
```
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: Please check comment above name of functions in this section.

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+							}
+						}
+					}
+	):Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+	end)
+	:Times(AtLeast(1))
+end
+
+function Test:RegisterApp1()
```
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: Please check comment above name of functions in this section.

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+		EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered")
+		:Do(function(_,data)
+			self.HMIAppID = data.params.application.appID
+		end)
+		self.mobileSession:ExpectResponse(correlationId, { success = true, resultCode = "SUCCESS" })
+		self.mobileSession:ExpectNotification("OnHMIStatus", {hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN"})
+	end)
+end
+
+function Test:PTU_revoke_app_group()
```
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: Please check comment above name of functions in this section.

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+			end)
+			self.mobileSession:ExpectResponse(CorIdSystemRequest, {success = true, resultCode = "SUCCESS"})
+			end)
+		end)
+	end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+commonFunctions:userPrint(34, "Test is intended to check that Policies Manager responds isAppPermissionRevoked:true for revoked app permissions")
```
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: Please remove this line, other tests do not contain so detailed information

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+		if data.result.isAppPermissionRevoked ~= true then
+			commonFunctions:userPrint(31, "Wrong SDL behavior: isAppPermissionRevoked should be false for app with revoked group")	
+			return false
+		else
+		    commonFunctions:userPrint(33, "isAppPermissionRevoked is true for app with revoked group - expected behavior")	
+		end	
+	end)	
+end
+
+--[[ Postconditions ]]
```
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: Please organize section --[[ Postconditions ]] as in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/98
The function in it should be last in your series. 
Also please note that names of functions in this section should contain PostCondition_

**Comment by maria-mih** [comment]
@vvvakulenko Please check comments related with template adherence.

**Comment by aarotar** [comment]
@maria-mih, please be more attentive when reviewing - you are addressing changes to wrong person, not to author of the script. 
This is confusing.

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+--			Make backup copy of preloaded PT
+--			Overwrite preloaded PT adding list of groups for specific app
+--			Connect device
+--			Register app
+--   		Revoke app group by PTU
+-- 			
+--     2. Performed steps
+-- 			Activate app
+--
+-- Requirement summary:
```
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: accepted

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+--     2. Performed steps
+-- 			Activate app
+--
+-- Requirement summary: 
+--    [Policies] SDL.ActivateApp from HMI and 'isAppPermissionsRevoked' parameter in the response
+--
+-- Expected result:
+--      PoliciesManager must respond with "isAppPermissionRevoked:true" and "AppRevokedPermissions" param containing the list of revoked permissions to HMI
+---------------------------------------------------------------------------------------------
+--[[ General Settings for configuration ]]
```
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: accepted

**Comment by maria-mih** [comment]
@aarotar Please accept my apologies, will be more attentive. 
Please check comments.

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua (hunk 8 above)
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: rejected, redundant variable for script.

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+Test = require('user_modules/connecttest_resumption')
+require('cardinalities')
+local mobile_session = require('mobile_session')
+
+--[[ General configuration parameters ]]
```
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: accepted

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
```
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: accepted

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+local Preconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
```
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: accepted

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua (hunk 1 above)
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: rejected, no need to duplicate Preconditions, this function is in block Preconditions. Redundant action.

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua (hunk 2 above)
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: rejected, no need to duplicate Preconditions, this function is in block Preconditions. Redundant action.

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua (hunk 3 above)
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: rejected, no need to duplicate Preconditions, this function is in block Preconditions. Redundant action.

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua (hunk 4 above)
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: rejected, no need to duplicate Preconditions, this function is in block Preconditions. Redundant action.

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua (hunk 5 above)
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: rejected. Does it have any impact on template? I guess this is not related to it.

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua (hunk 6 above)
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: accepted

**Comment by aarotar** [comment]
@maria-mih please see updated script in 7a69ad9.

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua (hunk 3 above)
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: Please check comment above, it is in template

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua (hunk 4 above)
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: Please check comment above, it is in template

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua (hunk 5 above)
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: The idea is to have uniform scripts, no other script so far has this section.

**Comment by maria-mih** [comment]
@aarotar Please check my answers.

File: test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua [hunk 12]
```diff
@@ -0,0 +1,153 @@
+---------------------------------------------------------------------------------------------
```
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua]: New comments

**Comment by aarotar** [comment]
@maria-mih, updated in last commit  91e748d.

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge.
